imagersite/templates/imager_profile/equipment_template.html

```diff
@@ -0,0 +1,9 @@
+{% extends "imagersite/base.html" %}
+{% block body %}
+
+    <h3>Type: {{ potographer.equipment.type_}}</h3>
```

Comment:
Also, according to the way in which you're including this template, you don't seem to need to include the photographer at all. You're already doing `for equip in photographer.equipment.all` so shouldn't you just be using `equip` here?